package.json

```diff
@@ -1,47 +1,53 @@
 {
-  "name": "dockerode",
-  "description": "Docker Remote API module.",
-  "version": "3.3.1",
-  "author": "Pedro Dias <petermdias@gmail.com>",
-  "maintainers": [
-    "apocas <petermdias@gmail.com>"
-  ],
+  "name": "sim-pipe-simulation-controller",
+  "version": "0.0.1",
+  "description": "Simulations and analytics for DataCloud pipelines.",
+  "exports": "./dist/index.js",
+  "type": "module",
+  "scripts": {
+    "start": "node --experimental-specifier-resolution=node --loader ts-node/esm/transpile-only ./src/index.ts",
+    "sftp-utils": "node --experimental-specifier-resolution=node --loader ts-node/esm/transpile-only ./src/sftp-utils.ts",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --ext .ts src",
+    "build": "tsc",
+    "start-compiled": "node --experimental-specifier-resolution=node ./dist/index.js"
+  },
+  "engines": {
+    "node": ">=16.0.0"
+  },
   "repository": {
     "type": "git",
-    "url": "http://github.com/apocas/dockerode.git"
+    "url": "git+https://github.com/DataCloud-project/SIM-PIPE-Simulation-Controller.git"
   },
-  "keywords": [
-    "docker",
-    "docker.io"
-  ],
-  "dependencies": {
-    "async": "^3.2.2",
-    "carrier": "^0.3.0",
-    "config": "^3.3.6",
-    "docker-cli-js": "^2.8.0",
-    "docker-modem": "^3.0.3",
-    "dockerode": "^3.3.1",
-    "dockerode-compose": "^1.2.2",
-    "fastify-cli": "^2.13.0",
-    "imagemin": "^8.0.1",
-    "npm": "^8.1.3",
-    "rb-strong-tunnel": "^1.2.3",
-    "ssh2": "^1.5.0",
-    "ssh2-sftp-client": "^7.2.0",
-    "tar-fs": "~2.0.1"
+  "author": "SINTEF",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/DataCloud-project/SIM-PIPE-Simulation-Controller/issues"
   },
+  "homepage": "https://github.com/DataCloud-project/SIM-PIPE-Simulation-Controller#readme",
   "devDependencies": {
-    "bluebird": "^3.7.0",
-    "chai": "~4.2.0",
-    "memorystream": "~0.3.0",
-    "mocha": "^9.1.3"
+    "@types/config": "^0.0.40",
+    "@types/dockerode": "^3.3.0",
+    "@types/node": "^16.11.13",
+    "@types/set-interval-async": "^1.0.0",
+    "@types/ssh2-sftp-client": "^7.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.7.0",
+    "@typescript-eslint/parser": "^5.7.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^16.1.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-simple-import-sort": "^7.0.0",
+    "eslint-plugin-unicorn": "^39.0.0",
+    "ts-node": "^10.4.0",
+    "typescript": "^4.5.4"
   },
-  "main": "./lib/docker",
-  "scripts": {
-    "test": "./node_modules/mocha/bin/mocha -R spec --exit"
-  },
-  "license": "Apache-2.0",
-  "engines": {
-    "node": ">= 8.0"
+  "dependencies": {
+    "config": "^3.3.6",
+    "dockerode": "^3.3.1",
+    "set-interval-async": "^2.0.3",
+    "ssh2-sftp-client": "^7.2.0"
```

Comment:
I fixed the `package.json` file. Now a single `npm install` is enough to have all the dependencies up and running.